data/sponsors.js

```diff
@@ -0,0 +1,69 @@
+import blizzard from '../public/images/sponsors/blizzard.png';
```

Comment:
maybe consider making an index.js file for the images so u can just do `import {blizzard, citadel, deloitee, etc.} from '../public/images/sponsors/';`

Reply:
Great idea, let's add it to the backlog 😄